platform/FileHandle.h

```diff
@@ -114,10 +114,10 @@ class FileHandle {
      *
      *  @return         Size of the file in bytes
      */
-    virtual size_t size()
+    virtual off_t size()
```

Comment:
what are the implications of this change (mbed 2 and mbed 2 filesystem or current mbed 5 filesystem users) ?